packages/bpk-component-image/src/BpkImage.js

```diff
@@ -56,22 +75,23 @@ Image.defaultProps = {
   hidden: false,
 };
 
-class BpkImage extends React.Component {
-  constructor(props) {
-    super(props);
+class BpkImage extends React.Component<BpkImageProps> {
+  onImageLoad: Function;
+  placeholder: any;
```

Comment:
Shouldn't this be `string`?

Reply:
Not a string, but an `HTMLElement` this is the same issue as https://github.com/Skyscanner/backpack/pull/431/files/6458d11cd9db3bc265558c454452a468f59a3a32#diff-be182dff2185fb989cfd339d396e50dfR163 `any` it's just a way around it